src/hooks/useWebAvatar.tsx

```diff
@@ -15,14 +15,26 @@ export function useWebAvatar() {
     if (!room) {
       return;
     }
+    if (room.state === ConnectionState.Connected) {
+      // support for <Avatar> instances created AFTER "connect" by making sure the new video and audio elements are attached
+      room.remoteParticipants.forEach((participant) => {
+        participant.trackPublications.forEach(({ track }) => {
+          handleTrackSubscribed(track!);
+        });
+      });
+    }
```

Comment:
nitpick (non-blocking): move this right before the return statement, I think it makes more sense to define the handlers and add the listeners before and then if already connected call them